.circleci/config.yml

```diff
@@ -94,12 +94,6 @@ commands:
           name: "Install awscli on macos machine"
           command: |
             brew install awscli
-  install-golangci-lint:
-    steps:
-      - run:
-          name: "Install golangci-lint"
-          command: |
-            curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.24.0
```

Comment:
v1.24.0 is only used here because of this line in `build` job.
https://github.com/klaytn/klaytn/blob/cb6bd9e50fe8cb640f860b8e9f20dd52efb36e91/.circleci/config.yml#L373

As far as I know, v1.24.0 is not working with linters used in klaytn code since it's too old.

It's great to remove this `install-golangci-lint`, but could you look at it at `make fmt`? I think it's not necessary anymore because `test-linter` already executing gofmt linter. Maybe removement of `doFmt` in ci.go file is needed.

Reply:
hmm, I'm not familiar with Go, maybe I need time to investigate that